lib/compat/experimental/blocks.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Temporary compatibility shims for features present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+if ( ! function_exists( 'build_comment_query_vars_from_block' ) ) {
+	/**
+	 * Helper function that constructs a comment query vars array from the passed block properties.
+	 *
+	 * It's used with the Comment Query Loop inner blocks.
+	 *
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return array Returns the comment query parameters to use with the WP_Comment_Query constructor.
+	 */
+	function build_comment_query_vars_from_block( $block ) {
```

Comment:
When adding unit tests we can replicate the corresponding functionality for the Query Loop block:

https://github.com/WordPress/wordpress-develop/blob/2648a5f984b8abf06872151898e3a61d3458a628/tests/phpunit/tests/blocks/wpBlock.php#L416-L472

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/37196/commits/8a8d0e2714ebdb247f6e48cb61845dd1ae5e9983. It's a simplified version as it turns out we can use the same approach that @michalczaplinski used for other test cases.